lychee-lib/src/collector.rs

```diff
@@ -1,32 +1,43 @@
-use crate::{
-    extract::{extract_links, FileType},
-    Request,
+use std::{
+    collections::HashSet,
+    fmt::Display,
+    path::{Path, PathBuf},
 };
-use anyhow::{anyhow, Context, Result};
+
 use glob::glob_with;
 use reqwest::Url;
 use serde::Serialize;
 use shellexpand::tilde;
-use std::path::Path;
-use std::path::PathBuf;
-use std::{collections::HashSet, fmt::Display};
-use tokio::fs::read_to_string;
-use tokio::io::{stdin, AsyncReadExt};
+use tokio::{
+    fs::read_to_string,
+    io::{stdin, AsyncReadExt},
+};
+
+use crate::{
+    extract::{extract_links, FileType},
+    Request, Result,
+};
 
 const STDIN: &str = "-";
 
+/// Links which to be validated.
```

Comment:
How about "Links which need to be validated" or "Links which should be validated"?

Reply:
Will fix in next commit with "Links which need to be validated"